sql/core/src/main/scala/org/apache/spark/sql/expressions/Window.scala

```diff
@@ -22,7 +22,7 @@ import org.apache.spark.sql.Column
 import org.apache.spark.sql.catalyst.expressions._
 
 /**
- * Utility functions for defining window in DataFrames.
+ * Utility functions for defining window in Datasets.
```

Comment:
Sure. I'm going to revert the changes. There's a little value in them. I'd rather see the changes approved in general than fight for DataFrame vs Dataset.

p.s. There's no `DataFrame` in Spark SQL which is just a type alias of `Dataset[Row]` -- see https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/package.scala#L46.